data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/ContainsSubstringExpressionFunction.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.opensearch.dataprepper.model.event.Event;
+import java.util.List;
+import javax.inject.Named;
+import java.util.function.Function;
+
+@Named
+public class ContainsSubstringExpressionFunction implements ExpressionFunction {
+    public String getFunctionName() {
+        return "containsSubstring";
+    }
+
+    public Object evaluate(final List<Object> args, Event event, Function<Object, Object> convertLiteralType) {
+        if (args.size() != 2) {
```

Comment:
Take it or leave it: `2` appears to be a repeated value here. I would recommend creating a constant in this class: `NUMBER_OF_ARGS`